From 3ac3b3e8fb6de5adceb96f90ca7b3a85a6885e39 Mon Sep 17 00:00:00 2001
From: Peter Krempa <pkrempa@redhat.com>
Date: Mon, 7 Feb 2022 11:03:19 +0100
Subject: qemuMonitorJSONGetCPUx86Data: Unexport
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The function is used only as a helper in src/qemu/qemu_monitor_json.c

Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/libvirt/+bug/1982896
Origin: upstream, https://gitlab.com/libvirt/libvirt/-/commit/3ac3b3e8fb6de5adceb96f90ca7b3a85a6885e39
Last-Update: 2023-08-04
---
diff --git a/src/qemu/qemu_monitor_json.c b/src/qemu/qemu_monitor_json.c
index b0b513683b..d81456bcad 100644
--- a/src/qemu/qemu_monitor_json.c
+++ b/src/qemu/qemu_monitor_json.c
@@ -7024,7 +7024,7 @@ qemuMonitorJSONParseCPUx86Features(virJSONValue *data)
 }
 
 
-int
+static int
 qemuMonitorJSONGetCPUx86Data(qemuMonitor *mon,
                              const char *property,
                              virCPUData **cpudata)
diff --git a/src/qemu/qemu_monitor_json.h b/src/qemu/qemu_monitor_json.h
index 64d9ebdaa3..be66c545c9 100644
--- a/src/qemu/qemu_monitor_json.h
+++ b/src/qemu/qemu_monitor_json.h
@@ -646,11 +646,6 @@ int
 qemuMonitorJSONGetDeviceAliases(qemuMonitor *mon,
                                 char ***aliases);
 
-int
-qemuMonitorJSONGetCPUx86Data(qemuMonitor *mon,
-                             const char *property,
-                             virCPUData **cpudata);
-
 int
 qemuMonitorJSONGetGuestCPUx86(qemuMonitor *mon,
                               virCPUData **data,
